src/manifest.json

```diff
@@ -10,8 +10,8 @@
   "incognito": "spanning",
   "permissions": [
     "tabs",
-    "http://*/*",
     "https://*/*",
+    "http://*/*",
```

Comment:
Let's avoid making any unnecessary manifest changes. This seems trivial, but who knows what buggy logic this array reordering may trigger.